### DISCUSSION TITLE
Question related to mini.jump2d

### DISCUSSION THREAD

**Discussion by thieung** [nvim-mini/mini.nvim#126] [Q&A] [answered]
Hi @echasnovski 
Thanks for making this awesome plugin.

Related to mini.jump2d, I would like to ask a concern that does it support jump within a region selection, like https://github.com/mfussenegger/nvim-treehopper does?

Thank you!

**Comment by echasnovski** [comment, marked as answer]
> Hi @echasnovski Thanks for making this awesome plugin.
> 
> Related to mini.jump2d, I would like to ask a concern that does it support jump within a region selection, like https://github.com/mfussenegger/nvim-treehopper does?
> 
> Thank you!

Hi!

I am not entirely sure what this plugin does about region selection. If it selects region based on tree-sitter nodes, then no: 'mini.jump2d' is meant as jumping to spots. For selecting custom regions there are [mini.ai](https://github.com/echasnovski/mini.nvim/blob/main/readmes/mini-ai.md) and incremental selection in 'nvim-treesitter' itself.

Jumping based on tree-sitter nodes, however, is an interesting idea and seems to be totally doable with a custom spotter. It is not a builtin, though. If this functionality is critical to you and you don't have time to implement custom spotter yourself, open an issue and I'll take a look at it when I get time.

**Reply by thieung** [reply]
Thanks @echasnovski
